packages/components/Button/src/Button.types.ts

```diff
@@ -166,14 +166,23 @@ export interface ButtonProps extends ButtonCoreProps {
   loading?: boolean;
 }
 
+interface ButtonState {
+  height?: number;
```

Comment:
should these be `measuredHeight` `measuredWidth` for variable-sized buttons e.g. localization?

Reply:
Sure, I can rename them.